chalice/chalicelib/aws/GdsAwsClient.py

```diff
@@ -0,0 +1,194 @@
+# GdsAwsClient
+# Manage sts assume-role calls and temporary credentials
+import os
+import boto3
+import json
+import configparser
+
+class GdsAwsClient:
+
+    # initialise empty dictionaries for clients and assume role sessions
+    resources = dict()
+    clients = dict()
+    sessions = dict()
+
+    # retrieve default key and secret from ~/.aws/credentials file
+    def load_credentials(self): 
+
+        # implement ini file parser for user home configuration file
+        config = configparser.ConfigParser()
+        home = os.getenv("HOME")
+        config.read(f"{home}/.aws/credentials")
```

Comment:
I thought we didn't need credentials there if we were using aws-valut, has this changed?